monitor/src/main/java/edu/asu/diging/monitor/core/model/impl/Group.java

```diff
@@ -0,0 +1,54 @@
+package edu.asu.diging.monitor.core.model.impl;
+
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Entity
+@Table(name = "`Group`")
```

Comment:
why are there extra `'`?

Reply:
The Group keyword is reserved in mysql. It throws an SQL syntax error if Group is used as the table name.